### ISSUE TITLE
United script failing on strptime

### ISSUE THREAD

**Issue by uda** [hasadna/anyway#684]
I got this on the new production server:
```
INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): www.ims.gov.il
DEBUG:requests.packages.urllib3.connectionpool:"GET /ims/PublicXML/observ.xml HTTP/1.1" 200 110970
INFO:root:Importing data from mail...
INFO:root:Login successful! Importing files, please hold...
Traceback (most recent call last):
  File "united.py", line 373, in <module>
    main()
  File "united.py", line 360, in main
    importmail.main(args.username, args.password, args.lastmail)
  File "/var/www/anyway/importmail.py", line 71, in main
    mtime = datetime.strptime(mail['Date'][:-6], '%a, %d %b %Y %H:%M:%S')
  File "/usr/lib/python2.7/_strptime.py", line 335, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains:  +0000
```

we need to check this script on a fresh installation

**Comment by uda** [comment]
I need this in order to add it to crontab as per #463

**Comment by r-darwish** [comment]
Wasn't this issue fixed in 0ef8eec?

**Comment by uda** [comment]
I ran the script yesterday and it seemed to work, I'll add it as a cron job
